ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -300,6 +301,15 @@ public boolean onTouchEvent(MotionEvent ev) {
     return super.onTouchEvent(ev);
   }
 
+  @Override
+  public boolean executeKeyEvent(KeyEvent event) {
+    int eventKeyCode = event.getKeyCode();
+    if (!mScrollEnabled && (eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP)) {
```

Comment:
Sorry, my comment about formatting was probably unclear. I didn't mean you should put all the checks on the same line, but the logical OR being at the end  rather that at the start of the line:

```
eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN || 
    eventKeyCode == KeyEvent.KEYCODE_DPAD_UP
```

instead of

```
eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN
    || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP

```